app/jobs/send_update_profile_reminder_mail_job.rb

```diff
@@ -0,0 +1,19 @@
+class SendUpdateProfileReminderMailJob < CronJob
+  self.cron_expression = '0 3 * * 7'
+
+  def perform
+    Person.find_each do |person|
+      company = Company.find(person.company_id)
+      if person.reminder_mails_active && company.reminder_mails_active && needs_reminder?(person)
+        ReminderMailer.update_user_reminder_email(person).deliver_now!
+      end
+    end
+  end
+
+  private
+
+  def needs_reminder?(person)
+    person.updated_at < 1.year.ago ||
+    person.people_skills.exists?(['updated_at < ?', 1.year.ago])
```

Comment:
Hmm this is a funny one.
The person gets updated via `associations_updated_at` for activities and such... which in turn causes and `updated_at` as well :grimacing: 

Maybe we should also use the `associations_updatet_at` for the people skills?